graylog2-server/src/main/java/org/graylog2/cluster/leader/StaticLeaderElectionService.java

```diff
@@ -43,4 +43,9 @@ public boolean isLeader() {
         //noinspection deprecation
         return configuration.isMaster();
     }
+
+    @Override
+    public void giveUpLeader() {
+        // Nothing we can do here
```

Comment:
It won't have an effect when we only call this upon shutdown. But I guess the following would theoretically be the correct thing to do:

```suggestion
        configuration.setIsMaster(false);
```